### DISCUSSION TITLE
Fix error when building LOS 19.1-20

### DISCUSSION THREAD

**Discussion by Sensei-Developer** [ROM-builders/temporary#19395] [Q&A]
Many have noticed that LOS has not been going to work lately, such an error comes out:

`FAILED: out/target/common/obj/APPS/webview_intermediates/enforce_uses_libraries.status
/bin/bash -c "(rm -f out/target/common/obj/APPS/webview_intermediates/enforce_uses_libraries.status ) && (build/soong/scripts/manifest_check.py     --enforce-uses-libraries     --enforce-uses-libraries-status out/target/common/obj/APPS/webview_intermediates/enforce_uses_libraries.status     --aapt out/host/linux-x86/bin/aapt                     --enforce-uses-libraries-relax     external/chromium-webview/prebuilt/arm64/webview.apk )"
zipro W 03-26 16:33:27 30038 30038] Error opening archive external/chromium-webview/prebuilt/arm64/webview.apk: Invalid file
ERROR: dump failed because no AndroidManifest.xml found
error: Command '['out/host/linux-x86/bin/aapt', 'dump', 'badging', 'external/chromium-webview/prebuilt/arm64/webview.apk']' returned non-zero exit status 1
`

I asked the LOS team how I could fix it, they told me that I needed to install the package due to their recent changes. Please do this so that we can collect LOS 19.1-20

`install git-lfs`

**Comment by Apon77** [comment]
sudo apt install git-lfs
This command?

**Reply by Sensei-Developer** [reply]
Yes, sir
